internal/plumbing/tree_diff.go

```diff
@@ -96,7 +102,12 @@ func (treediff *TreeDiff) ListConfigurationOptions() []core.ConfigurationOption
 			"which disables this filter and lets all the files through.", allLanguages),
 		Flag:        "languages",
 		Type:        core.StringsConfigurationOption,
-		Default:     []string{allLanguages}},
+		Default:     []string{allLanguages} }, {
+	  Name:       ConfigTreeDiffFilterRegex,
+    Description: "Will filter which files to analyze based on this Regex",
+		Flag:        "filter",
```

Comment:
How about naming this `whitelist`

Reply:
I originally had this as whitelisted-prefixes.  Felt a little odd and felt that regexes could work better.  

I'm fine with switching this to just whitelist.